Gems/AWSGameLift/Code/AWSGameLiftServer/Source/AWSGameLiftServerManager.cpp

```diff
@@ -406,8 +406,30 @@ namespace AWSGameLift
 
     AZ::IO::Path AWSGameLiftServerManager::GetExternalSessionCertificate()
     {
-        // TODO: Add support to get TLS cert file path
-        return AZ::IO::Path();
+        auto certificateOutcome = m_gameLiftServerSDKWrapper->GetInstanceCertificate();
+        if (certificateOutcome.IsSuccess())
+        {
+            return AZ::IO::Path(certificateOutcome.GetResult().GetCertificatePath().c_str());
+        }
+        else
+        {
+            AZ_Warning(AWSGameLiftServerManagerName, false, AWSGameLiftServerInstanceCertificateErrorMessage);

```

Comment:
As this is a fleet level option, is this going to cause false alarms on fleets without fleet certs? This would only be an error if we expect the fleet to be setup with certs, otherwise theres the behaviour is as expected and a warning is not useful.

Reply:
Make sense, I can add a check inside networking component to make sure this function won't be triggered for non tls use case.

Reply:
done